src/Context/Model/CheckoutContext.php

```diff
@@ -138,14 +107,22 @@ private function getSettings(): array
      */
     private function getStrings(): array
     {
-        return array_merge(
-            Language::translateArray(self::TRANSLATION_MAP),
-            [
-                'headerDeliveryOptions' => Settings::get(
-                    CheckoutSettings::DELIVERY_OPTIONS_HEADER,
-                    CheckoutSettings::ID
-                ),
-            ]
-        );
+        $prefix  = Pdk::get('translationPrefixDeliveryOptions');
+        $strings = (new Collection(Language::getTranslations()))
+            ->filter(static function ($value, $key) use ($prefix) {
+                return Str::startsWith($key, $prefix);
+            })
+            ->map([Language::class, 'translate'])
+            ->mapWithKeys(static function ($value, $key) use ($prefix) {
+                $replacedKey = Str::after($key, $prefix);
+                $finalKey    = Str::camel($replacedKey);
```

Comment:
Je kunt deze eventueel ook inlinen als 

```php
return [Str::camel(Str::after($key, $prefix)) => $value];
```

Dat is misschien wat minder leesbaar, maar wel minder functioneel overbodige variables

Reply:
Ik ging idd voor leesbaarheid :)